### PR TITLE
feat: support bootstrap image mirror in China

### LINKED ISSUE #80: bootstrap image for china region
ECR public for China regions may be slow occasionally

https://github.com/pahud/cdk-keycloak/blob/f43879a6e830a094e97eae19b5ce095dd66c8e00/src/keycloak.ts#L436

we should consider to use nwcd mirror at:

https://github.com/nwcdlabs/container-mirror/blob/dacc9afac86ad42ee133bbd6fa1a56a68d6d94ba/mirror/required-images-daily.txt#L13

### PR DESCRIPTION
**Pull request by pahud**
Fixes #80

### CODE REVIEW

**Comment by pahud** [comment]
Beijing region is working

圖片

File: src/keycloak.ts [hunk 1]
```diff
@@ -541,23 +551,29 @@ export class ContainerService extends cdk.Construct {
       props.database.connections.allowDefaultPortFrom(bast);
     }
   }
-  private getKeyCloakDockerImageUri(): string {
+  private getImageUriFromMap(map: dockerImageMap, cfnMapping: string): string {
```
Comment (wchaws) [inline src/keycloak.ts:554]: How about rename `cfnMapping` to `id` ?

Comment (pahud) [reply src/keycloak.ts:554]: @wchaws OK

**Comment by pahud** [comment]
Ningxia is doing good as well

圖片

**Comment by pahud** [comment]
testing the Tokyo region now.

**Comment by pahud** [comment]
Tokyo is doing good as well

圖片


I think it's ready to merge now. @wchaws

**Review by wchaws — APPROVED** [review]
LGTM

**Pull request merged**